SPECS/python-pytest-cov/python-pytest-cov.spec

```diff
@@ -0,0 +1,163 @@
+%global srcname pytest-cov
+
+Summary:        Pytest plugin for coverage reporting
+Name:           python-%{srcname}
+Version:        2.10.1
+Release:        3%{?dist}
+License:        MIT
+URL:            https://pypi.python.org/pypi/%{srcname}
+Vendor:         Microsoft
+Distribution:   Mariner
+#Source0:       https://github.com/pytest-dev/%{srcname}/archive/v%{version}/%{srcname}-%{version}.tar.gz
+Source0:        https://github.com/pytest-dev/%{srcname}/archive/v%{version}/%{name}-%{version}.tar.gz
```

Comment:
>https://github.com/pytest-dev/%{srcname}/archive/v%{version}/ [](http://example.com/codeflow?start=16&length=61)

If this URL exists, we can delete the commented our line above. If it doesn't exist, we should leave this line as just %{name}-%{version}.tar.gz